README.md

```diff
@@ -71,3 +75,18 @@ a suitable alert can be setup for persistent differences:
 These alerts can be sent to Slack, for example:
 
 ![Slack Alert](/imgs/alert.png)
+
+## compare-images
+
+To quickly see how two environments differ, in terms of images:
```

Comment:
The term environment is poorly defined, you may want to call these "sets of configurations"